R/stat-bin.r

```diff
@@ -9,15 +9,15 @@
 #'   bin width of a time variable is the number of seconds.
 #' @param bins Number of bins. Overridden by `binwidth`. Defaults to 30.
 #' @param center The center of one of the bins.  Note that if center is above or
-#'   below the range of the data, things will be shifted by an appropriate
-#'   number of `width`s. To center on integers, for example, use
+#'   below the range of the data, things will be shifted by the appropriate
```

Comment:
It's supposed to imply that it will be shifted by an integer multiple of widths to lie within the range of the data. I'm not sure how to say that better

Reply:
Might be easier to document `center` and `boundary` together?